src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+
+ReactDOM.render(
```

Comment:
You should use `ReactDOM.createRoot` instead https://reactjs.org/docs/react-dom-client.html